web/src/styles/MapView.css

```diff
@@ -15,4 +15,5 @@
 .maplibregl-ctrl-top-left,
 .maplibregl-ctrl-top-right {
   z-index: 0;
+  margin-bottom: 48px;
```

Comment:
❓ can you just give me hint why you added this margin

Reply:
This fixes the overlap of footer and zoom buttons on pois desktop with smaller (but not yet mobile) screen widths, see e.g. here:
![image](https://github.com/user-attachments/assets/ea559cf2-a730-4bb0-aad9-d76d11f145a8)